.gitignore

```diff
@@ -360,4 +360,5 @@ build/
 dist/
```

Comment:
Can you please exclude this file from the PR?

Reply:
Sure. Done, thank you